project_resource_calendar/models/instrument.py

```diff
@@ -27,3 +27,15 @@ class Instrument(models.Model):
         index=True,
         default='instrument',
     )
+    instrument_log_count = fields.Integer(
+        string='Calendar Logs',
+        compute='_compute_instrument_log_count',
+    )
+
+    def _compute_instrument_log_count(self):
+        for rec in self:
+            rec.instrument_log_count = self.env['auditlog.log'].search_count([
```

Comment:
This is repeated code, this function is used on multiple modules, you should create new class and inherit

Reply:
Is it possible?

Reply:
@eilst As we discuss , I have created the test cases for calendar event modules. I will be sharing the Task number to you. Please merge this task